docs/development.rst

```diff
@@ -1,17 +0,0 @@
-Development
```

Comment:
I still work that way. What do you do ?

Reply:
The information in `development.rst` was a duplicate of what is in `contributing.rst` ... or did I miss something?

Reply:
Yes you are right.